src/Aspire.ProjectTemplates.9.0.net9/templates/aspire-apphost/.template.config/template.json

```diff
@@ -0,0 +1,221 @@
+{
+  "$schema": "https://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [
+    "Common",
+    ".NET Aspire",
+    "Cloud"
+  ],
+  "name": ".NET Aspire App Host",
+  "defaultName": "AppHost",
+  "description": "A project template for creating a .NET Aspire app host (orchestrator) project.",
+  "shortName": "aspire-apphost",
+  "sourceName": "Aspire.AppHost1",
+  "preferNameDirectory": true,
+  "tags": {
+    "language": "C#",
+    "type": "project"
+  },
+  "precedence": "9000",
+  "identity": "Aspire.AppHost.CSharp.9.net9",
+  "thirdPartyNotices": "https://aka.ms/dotnet/aspire/8.0-third-party-notices",
+  "groupIdentity": "Aspire.AppHost.9",
+  "guids": [
+    "98048C9C-BF28-46BA-A98E-63767EE5E3A8"
+  ],
+  "symbols": {
+    "Framework": {
+      "type": "parameter",
+      "description": "The target framework for the project.",
+      "datatype": "choice",
+      "choices": [
+        {
+          "choice": "net9.0",
+          "description": "Target net9.0"
+        }
+      ],
+      "replaces": "net8.0",
```

Comment:
Why is a replaces needed here? Shouldn't these templates all be targeting `net9.0` anyway? Or does this control something else?

Reply:
I think we want to keep this, but make it net9.0 (as well as changing the csproj file as well) - this would make the authoring consistent with other 1st party templates, including aspnetcore.

Reply:
done.